.editorconfig

```
@@ -10,8 +10,8 @@ trim_trailing_whitespace = true
 
 [*.{c,h}]
 indent_style = tab
-# We specifically don't set indent_size and tab_with here so people can choose
-# what they prefer themselves
+indent_size = tab
+tab_width = 8
```

I took a bit of time to actually read the editorconfig docs on the difference between `indent_size` and `tab_width`. And it turns out you only need to set `indent_size` here:

> tab_width: a whole number defining the number of columns used to represent a tab character. This defaults to the value of indent_size and doesn't usually need to be specified.

```suggestion
indent_size = 8
```

---

Done, and also removed the other tab_width settings that don't seem necessary.